cli/src/main/kotlin/commands/ScannerCommand.kt

```diff
@@ -128,7 +128,8 @@ object ScannerCommand : CommandWithHelp() {
         println("Using scanner '$scanner'.")
 
         val ortResult = ortFile?.let {
```

Comment:
commit message nits:

missing a word: require to explicitly `specify` it ?
"This is necessary to known": maybe "It is necessary to know..."

consider using instead: The directory must be known beforehand in order to check whether the target files do already exist.